pipeline/extract.py

```diff
@@ -164,6 +174,8 @@ def parse_pdf(court_case: dict):
 
         case_no = re.search(
```

Comment:
Nice expression for grabbing the case number ids, better than web scraping directly!